doc/faq.rst

```diff
@@ -0,0 +1,199 @@
+Frequently Asked Questions
+##########################
+
+General
+-------
+
+.. _how_does_looking_glass_work:
+
+How does Looking Glass work?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please see the following video that covers this:
+
+https://www.youtube.com/watch?v=U44lihtNVVM
+
+.. _can_i_feed_the_vm_directly_into_obs:
+
+Can I feed the VM directly into OBS?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Looking Glass now features a functional :doc:`OBS plugin <obs>`, which acts as
+another Looking Glass client, but instead gives the captured frames to OBS.
+
+.. _why_is_my_ups_updates_per_second_so_low:
+
+Why is my UPS (Updates Per Second) so low?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are several reasons why this can happen, the most common is your
+capture resolution. At the time of this writing, the windows capture
+methods struggle to capture high resolutions under certain
+circumstances, as a general rule of thumb anything greater than 1200p
+(1920x1200) starts to suffer from performance issues.
```

Comment:
Rule of thumb feels outdated, but maybe that's out of scope for this PR.

Reply:
not necessarily

Reply:
The best thing I think is providing a look up table for all the common resolutions and tell people to round up.

Reply:
my bad, i should have specified that I was replying to

> maybe that's out of scope for this PR.

a lookup table can be added later